st/clitests/awsiam_spec.py

```diff
@@ -137,6 +137,18 @@ def iam_policy_crud_tests():
     #get-policy
     AwsIamTest('Get Policy').get_policy(arn).execute_test().command_response_should_have("iampolicy")
 
+    #get-policy-version
+    AwsIamTest('Get Policy Version').get_policy_version(arn, "v1").execute_test() \
+        .command_is_successful().command_response_should_have("POLICYVERSION")
+
+    #get-policy-version with wrong version id
+    AwsIamTest('Get Policy Version').get_policy_version(arn, "v4").execute_test(negative_case=True) \
+        .command_should_fail().command_error_should_have("NoSuchEntity")
+
+    #get-policy-version with wrong policy arn
+    AwsIamTest('Get Policy Version').get_policy_version("arn:seagate:iam::893266879798:policy/iampolicy67", "v1").execute_test(negative_case=True) \
```

Comment:
Correct arn here.It should not contain aws:seagate
It should be arn:aws....
Right now, This will fail in parameter validation only.

Reply:
done. Please check